WalletWasabi.Tests/UnitTests/Blockchain/Keys/WpkhOutputDescriptorHelperTests.cs

```diff
@@ -23,4 +23,20 @@ public void BasicTest()
 		Assert.Equal("wpkh([f3a4c42f/84'/0'/0']tprv8ghYQhz7XQhoqDZG8SzbkqGCDTwAzyVVmUN3cUerPhUgK91Xvc4FaMJpYwrjuQ48WD7KdQ7Y6znKnaY9PXP8SiDLv1srjjs8NVYGuM7Hrrk/1/*)#ktc4yfd7", descriptors.PrivateInternal);
 		Assert.Equal("wpkh([f3a4c42f/84'/0'/0']tprv8ghYQhz7XQhoqDZG8SzbkqGCDTwAzyVVmUN3cUerPhUgK91Xvc4FaMJpYwrjuQ48WD7KdQ7Y6znKnaY9PXP8SiDLv1srjjs8NVYGuM7Hrrk/0/*)#8la5euax", descriptors.PrivateExternal);
 	}
+
+	[Fact]
+	public void VerifyGenerateCorrectScript()
+	{
+		var km = KeyManager.CreateNew(out _, "", Network.Main);
+		HDFingerprint masterFingerprint = new(0x2fc4a4f3);
+
+		var watchOnlyKm = KeyManager.CreateNewHardwareWalletWatchOnly(masterFingerprint, km.ExtPubKey, Network.Main);
```

Comment:
Is there a faster way to get a functioning `ExtPubKey`? There must be.

Reply:
What do you mean by faster? More performant or shorter? Here you have an alternative:

```c#
var extPubKey = new ExtKey().Neuter();
```

Reply:
Thanks, this is what i was looking for. Creating a `KeyManager` only to get a `ExtPubKey` felt wrong.